src/main/java/com/ait/lienzo/client/core/shape/growable/ScrollStateDragHandlerFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018 Ahome' Innovation Technologies. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ait.lienzo.client.core.shape.growable;
+
+import com.ait.lienzo.client.core.shape.wires.WiresShape;
+
+public class ScrollStateDragHandlerFactory {
+
+    public static IScrollStateDragHandler create(Object shape,
+                                                 GrowableLienzoPanel growableLienzoPanel,
+                                                 GrowableLienzoScrollState scrollState) {
+
+        if (shape instanceof WiresShape) {
```

Comment:
Since ```shape``` is only used here; it might be clearer to move the check to [here](https://github.com/kiegroup/lienzo-core/pull/22/files#diff-ed6008c5d9e62fb522160b5c3af21d28R68) and don't throw an exception if the ```shape``` is not a ```WiresShape```. Simply silently ignoring will probably be better IMO.